color.html

```diff
@@ -3,14 +3,16 @@
 <custom-style>
   <style>
     html {
+      --material-base-color: #fff;
+      --dark-theme-base-color: #000;
```

Comment:
The base color is the same as `--material-body-background-color`, we should use/rename that instead.

I would like to avoid introducing another color just because of the dark theme variation of the split layout. So please remove `--dark-theme-base-color`, and let’s add a split-layout specific tweak instead. We can re-evaluate the need if some other elements needs a similar kind of color.